### ISSUE TITLE
Unable to launch fsmapper.exe

### ISSUE THREAD

**Issue by tp6m35p4** [opiopan/fsmapper#3]
After I switch to the newest commit. I successfully built it but fail to launch it.

OS: Windows11
IDE and compiler: VS 2022

The followings are the error message I got from VS 2022 debug console. I tried to run it in debug mode and set GUI project as startup project. I am not sure if it is correct to debug all projects. 
Thanks again for answering my question and creating this project!!

```
'fsmapper.exe' (Win32): Loaded 'C:\Users\ycchen\code\fsmapper\src\x64\Debug\fsmapper.exe'. Symbols loaded.
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\ntdll.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\kernel32.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\KernelBase.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\user32.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\win32u.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\gdi32.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\gdi32full.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\msvcp_win.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\ucrtbase.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\shell32.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\ole32.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Users\ycchen\code\fsmapper\src\x64\Debug\fsmappercore.dll'. Symbols loaded.
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\dwmapi.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\combase.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\combase.dll'. 
'fsmapper.exe' (Win32): Unloaded 'C:\Windows\System32\combase.dll'
'fsmapper.exe' (Win32): Loaded 'C:\Users\ycchen\code\fsmapper\src\x64\Debug\Microsoft.WindowsAppRuntime.Bootstrap.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\rpcrt4.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\oleaut32.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\msvcp140d.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\advapi32.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\msvcrt.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\vcruntime140d.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\vcruntime140_1d.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\sechost.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\ucrtbased.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Users\ycchen\code\fsmapper\src\x64\Debug\SimConnect.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\WinSxS\amd64_microsoft.windows.gdiplus_6595b64144ccf1df_1.1.22000.675_none_ce846f9a12fab040\GdiPlus.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\d2d1.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\d3d10_1.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\ws2_32.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\shlwapi.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Users\ycchen\code\fsmapper\src\x64\Debug\fsmapperhook.dll'. Symbols loaded.
'fsmapper.exe' (Win32): Loaded 'C:\Users\ycchen\code\fsmapper\src\x64\Debug\vJoyInterface.dll'. Module was built without symbols.
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\dinput8.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\setupapi.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\imagehlp.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\d3d11.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\d3d10_1core.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\vcruntime140.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\msvcp140.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\hid.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\vcruntime140_1.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\dxgi.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\imm32.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\InputHost.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\CoreMessaging.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\windows.storage.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\WinTypes.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\uxtheme.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\SHCore.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\bcryptprimitives.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\kernel.appcore.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\cryptbase.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\clbcatq.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\msctf.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\AppExtension.dll'. 
'fsmapper.exe' (Win32): Loaded 'C:\Windows\System32\Windows.StateRepositoryPS.dll'. 
D:\a\_work\1\s\dev\WindowsAppRuntime_BootstrapDLL\MddBootstrap.cpp(469)\Microsoft.WindowsAppRuntime.Bootstrap.dll!00007FF8B8B52F4F: (caller: 00007FF8B8B5225D) Exception(1) tid(2fe1c) 80070491 There was no match for the specified key in the index.
    Msg:[AppExtension.Name=microsoft.winappruntime.ddlm-1.0-x6, Major=1, Minor=0, Tag=, MinVersion=0.319.455.0] 
D:\a\_work\1\s\dev\WindowsAppRuntime_BootstrapDLL\MddBootstrap.cpp(125)\Microsoft.WindowsAppRuntime.Bootstrap.dll!00007FF8B8B63D61: (caller: 00007FF62E32410E) ReturnHr(1) tid(2fe1c) 80070491 There was no match for the specified key in the index.
    Msg:[D:\a\_work\1\s\dev\WindowsAppRuntime_BootstrapDLL\MddBootstrap.cpp(469)\Microsoft.WindowsAppRuntime.Bootstrap.dll!00007FF8B8B52F4F: (caller: 00007FF8B8B5225D) Exception(1) tid(2fe1c) 80070491 There was no match for the specified key in the index.
    Msg:[AppExtension.Name=microsoft.winappruntime.ddlm-1.0-x6, Major=1, Minor=0, Tag=, MinVersion=0.319.455.0] 
] 
Exception thrown at 0x00007FF947EE474C in fsmapper.exe: Microsoft C++ exception: wil::ResultException at memory location 0x0000001584EFCAA0.
Exception thrown at 0x00007FF947EE474C in fsmapper.exe: Microsoft C++ exception: [rethrow] at memory location 0x0000000000000000.
The thread 0x2e490 has exited with code 2147943569 (0x80070491).
The thread 0x2d188 has exited with code 2147943569 (0x80070491).
The thread 0x2eab0 has exited with code 2147943569 (0x80070491).
The thread 0x2b2c8 has exited with code 2147943569 (0x80070491).
The thread 0x2ec40 has exited with code 2147943569 (0x80070491).
The thread 0x2fea8 has exited with code 2147943569 (0x80070491).
The thread 0x2eaa4 has exited with code 2147943569 (0x80070491).
The thread 0x2d91c has exited with code 2147943569 (0x80070491).
The thread 0x2fee0 has exited with code 2147943569 (0x80070491).
The program '[183300] fsmapper.exe' has exited with code 2147943569 (0x80070491).
```

**Comment by opiopan** [comment]
It seems something happens  at the starting Windows App SDK bootstrap process.
Could you try to reinstall latest 1.0 version line of Windows SDK Runtime.
Please note that reinstall the runtime package, not VS extension

**Comment by pietia336** [comment]
Hi, I have exactly the same issue. Reinstalling the Windows App SDK Runtime did not help. 
I get the following error:
`<EventData>
  <Data Name="Bucket">1875521772481503259</Data> 
  <Data Name="BucketType">5</Data> 
  <Data Name="EventName">BEX64</Data> 
  <Data Name="Response">Niedostępny</Data> 
  <Data Name="CabId">0</Data> 
  <Data Name="P1">fsmapper.exe</Data> 
  <Data Name="P2">0.9.4.0</Data> 
  <Data Name="P3">65f5a739</Data> 
  <Data Name="P4">Microsoft.WindowsAppRuntime.Bootstrap.dll</Data> 
  <Data Name="P5">0.0.0.0</Data> 
  <Data Name="P6">62a3ddc7</Data> 
  <Data Name="P7">000000000000124d</Data> 
  <Data Name="P8">c0000409</Data> 
  <Data Name="P9">0000000000000007</Data> 
  <Data Name="P10" /> 
  <Data Name="AttachedFiles">\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.795c99e1-eb83-4409-a173-8ac3398c88bd.tmp.dmp \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.808ee6a7-5ffd-4e1a-85a3-cc9f064e7f53.tmp.WERInternalMetadata.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.fda2bd00-67b6-4972-9e3a-29120545dede.tmp.csv \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.2aaa9452-c559-4842-b010-3b680444de3f.tmp.txt \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.26d90cd2-b225-497b-b3c7-b659761ece83.tmp.xml</Data> 
  <Data Name="StorePath">\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_fsmapper.exe_bb4eb54ac363d4dbce74acfbe3b77fad6792b50_2df8e995_3ae5c3bf-ba36-4c0e-9aa5-98a3edd59e14</Data> 
  <Data Name="AnalysisSymbol" /> 
  <Data Name="Rechecking">0</Data> 
  <Data Name="ReportId">9f31ce55-8339-4176-a631-67c75fd88975</Data> 
  <Data Name="ReportStatus">268435456</Data> 
  <Data Name="HashedBucket">d37604782aaa4058aa07311d7945601b</Data> 
  <Data Name="CabGuid">0</Data> 
  </EventData>`
Please help.

**Comment by opiopan** [comment]
Thank you for reporting. It seems that updating the Windows App SDK as you suggested in another email resolves this issue.

**Comment by opiopan** [comment]
Closed this as completed in 3890397
